### PR TITLE
app-admin/doas: run commands as super user or another user, alternative to sudo from OpenBSD

### PR DESCRIPTION
**Pull request by doughdemon**
CC @gentoo/proxy-maint. You can find a build log for arm at: https://bpaste.net/show/6fefd69b86de

### CODE REVIEW

File: app-admin/doas/doas-6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+src_prepare()
+{
+	sed -i 's/-Werror //' Makefile || die
+	default
+}
+
+src_configure()
+{
+	tc-export CC AR
+	./configure \
```
Comment (monsieurp) [inline app-admin/doas/doas-6.0.ebuild]: Why not using `econf`?

File: app-admin/doas/doas-6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Run commands as super user or another user, alternative to sudo from OpenBSD"
+
+MY_PN=OpenDoas
+MY_P=${MY_PN}-${PV}
+HOMEPAGE="https://github.com/Duncaen/OpenDoas"
+SRC_URI="https://github.com/Duncaen/${MY_PN}/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
```
Comment (monsieurp) [inline app-admin/doas/doas-6.0.ebuild]: `v${PV}.tar.gz -> ${P}.tar.gz` so you don't need to tinker with `${S}` later on.

**Comment by doughdemon** [comment]
Thanks for the review.

> Why not using `econf`?

Because the configure script is not from autoconf, and would complain
about the unknown `--infodir` option.

> +MY_PN=OpenDoas
> +MY_P=${MY_PN}-${PV}
> +HOMEPAGE="https://github.com/Duncaen/OpenDoas"
> +SRC_URI="https://github.com/Duncaen/${MY_PN}/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
> 
> `v${PV}.tar.gz -> ${P}.tar.gz` so you don't need to tinker with `${S}` later on.

I had previously used this. However note that the tarball unpacks as
OpenDoas-6.0. So tinkering with `${S}` is necessary anyway. As it is
now the tarball name matches the directory it unpacks to.

**Comment by monsieurp** [comment]
OK, fine. Let's merge it.

**Comment by doughdemon** [comment]
monsieurp wrote:
> OK, fine. Let's merge it.

thanks!

**Pull request closed without merging**